generators/chipyard/src/main/scala/SpikeTile.scala

```diff
@@ -0,0 +1,392 @@
+package chipyard
+
+import chisel3._
+import chisel3.util._
+import chisel3.experimental.{IntParam, StringParam, IO}
+
+import freechips.rocketchip.config._
+import freechips.rocketchip.subsystem._
+import freechips.rocketchip.devices.tilelink._
+import freechips.rocketchip.diplomacy._
+import freechips.rocketchip.rocket._
+import freechips.rocketchip.tilelink._
+import freechips.rocketchip.interrupts._
+import freechips.rocketchip.util._
+import freechips.rocketchip.tile._
+import freechips.rocketchip.prci.ClockSinkParameters
+
+case class SpikeCoreParams(
+  val maxInsnsPerCycle: Int = 10000
```

Comment:
What's this used for?

Reply:
How many cycles spike can execute in a single "tick". (If spike sees a cache miss, however, the actual IPC will be lower).